db/change_cache.go

```diff
@@ -646,8 +646,8 @@ func (c *changeCache) processEntry(change *LogEntry) base.Set {
 		// There's a missing sequence (or several), so put this one on ice until it arrives:
 		heap.Push(&c.pendingLogs, change)
 		numPending := len(c.pendingLogs)
-		base.Infof(base.KeyCache, "  Deferring #%d (%d now waiting for #%d...#%d)",
-			sequence, numPending, c.nextSequence, c.pendingLogs[0].Sequence-1)
+		base.Infof(base.KeyCache, "  Deferring #%d (%d now waiting for #%d...#%d) doc %q / %q",
```

Comment:
The change doesn't necessarily have a docid and revid (could be principal, could be unused sequence).  What additional value do you expect from knowing the doc/rev here?

Reply:
Fair point, but this is consistent with other logging:

```
2019-01-08T13:56:20.086Z [INF] Cache: Received #136957696 (unused sequence)
2019-01-08T13:56:20.629Z [INF] Cache:   Received previously skipped out-of-order change (seq 136990108, expecting 137056689) doc "my_doc" / "2-aaaaaaaaaaaaaaaaaaaaaaaaaaaa"
2019-01-08T13:56:20.954Z [INF] Cache:   Received previously skipped out-of-order change (seq 136957696, expecting 137056689) doc "" / ""
```

As for value, I think this largely falls under the grep use case - it makes it easy to pull out all the events for a given doc/rev/seq.

Reply:
Ok - that's reasonable.